clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java

```diff
@@ -59,6 +58,14 @@ public KubernetesCredentials(KubernetesApiAdaptor apiAdaptor,
 
       for (int j = 0; j < affectedNamespaces.size(); j++) {
```

Comment:
Since you're not using the index elsewhere, why not use a for-each loop here?
`for (String inNamespace : affectedNamespaces)`